### DISCUSSION TITLE
underground geometry

### DISCUSSION THREAD

**Discussion by lavi-k** [pelicanmapping/osgearth#2045] [Support Q&A]
I am trying to make the earth transparent by setting blending property "true" in osgearth 3.1 so I can see underground geometry but its not working please let me know how I can do that or have an example please share with me

**Comment by gwaldron** [comment]
In osgEarth 3.1 you may need to force the terrain to draw after the geometry. You can make the earth semi-transparent and make it draw last like this:
```xml
<map>
   <options>
      <terrain color="#ffffff7f" bin_number="12"/>
```
The `bin_number` refers to an OSG render bin number. By default, feature data draws in bin number 10 (OSG's depth-sorted render bin).

**Reply by lavi-k** [reply]
- Thank @gwaldron,  now able to get a semi-transparent/transparent globe on changing the color to #ffffff00
- Unfortunately, when the geometry is complete behind the earth the geometry suddenly disappears.
- I suspect it is some sort of horizon culling. Is there a way to turn that off?
- See video, notice @ 0.6

https://user-images.githubusercontent.com/102519019/164702766-7fa60c83-1930-49f8-a9c2-216bd9a6e8a7.mp4

 seconds
